.github/labels.yml

```diff
@@ -66,6 +66,14 @@
     color: '#91e4e5'
     aliases: [type-shell-xonsh]
 
+# OS
+  - name: os::cygwin
+    description: pertains to Cygwin, commonly run on Windows
+    color: '#d0e0e3'
+  - name: os::msys2
+    description: pertains to MSYS2, commonly run on Windows
+    color: '#d0e0e3'
```

Comment:
Are we making them the same color on purpose?

Reply:
No, it's because that's the same color that's used for the other "OS" labels in the conda/infrastructure repository:

https://github.com/conda/infrastructure/blob/9bec3e981c5a4f41a37ee667084fd005e6d3ab62/.github/global.yml#L223-L248

Reply:
okay, got it